tests/acceptance/guid-file/registration-file-detail-test.ts

```diff
@@ -98,7 +118,196 @@ module('Acceptance | guid file | registration files', hooks => {
+        await click('[data-test-cancel-editing-metadata-button]');
+        assert.dom('[data-test-edit-metadata-form]').doesNotExist();
+        await click('[data-test-edit-metadata-button]');
+
+        // Screenshot before changes
+        await percySnapshot(assert);
+        // Update title
+        const editTitleTextArea = document.querySelectorAll('textarea')[0];
+        editTitleTextArea.textContent = 'A test title.';
+        assert.dom('[data-test-title-field]')
+            .containsText('A test title.', 'Metadata title field updates.');
+        // Update description
+        const editDescriptionTextArea = document.querySelectorAll('textarea')[1];
+        editDescriptionTextArea.textContent = 'A test description.';
+        assert.dom('[data-test-description-field]')
+            .containsText('A test description.', 'Metadata description field updates.');
+        // Update resource type
+        await selectChoose('[data-test-select-resource-type]', 'InteractiveResource');
+        // Update resource language
+        await selectChoose('[data-test-select-resource-language]', 'Abkhazian');
+        await selectChoose('[data-test-select-resource-language]', 'Arabic');
+        await selectChoose('[data-test-select-resource-language]', 'Chinese');
+        await selectChoose('[data-test-select-resource-language]', 'English');
+        await selectChoose('[data-test-select-resource-language]', 'French');
+        await selectChoose('[data-test-select-resource-language]', 'German');
+        await selectChoose('[data-test-select-resource-language]', 'Hindi');
+        await selectChoose('[data-test-select-resource-language]', 'Italian');
+        await selectChoose('[data-test-select-resource-language]', 'Japanese');
+        await selectChoose('[data-test-select-resource-language]', 'Korean');
+        await selectChoose('[data-test-select-resource-language]', 'Latin');
+        await selectChoose('[data-test-select-resource-language]', 'Multiple languages');
+        await selectChoose('[data-test-select-resource-language]', 'North American Indian languages');
+        await selectChoose('[data-test-select-resource-language]', 'Polish');
+        await selectChoose('[data-test-select-resource-language]', 'Portuguese');
+        await selectChoose('[data-test-select-resource-language]', 'Russian');
+        await selectChoose('[data-test-select-resource-language]', 'South American Indian languages');
+        await selectChoose('[data-test-select-resource-language]', 'Spanish');
+        await selectChoose('[data-test-select-resource-language]', 'Swedish');
+        await selectChoose('[data-test-select-resource-language]', 'Tagalog');
+        await selectChoose('[data-test-select-resource-language]', 'Thai');
+        await selectChoose('[data-test-select-resource-language]', 'Vietnamese');
+        await selectChoose('[data-test-select-resource-language]', 'Zuni');
```

Comment:
Out of curiosity, why did you select 23 different languages here?

Reply:
It was the most complete without testing edge cases for content uploaded. If I have missed any you would like to see included, please let me know.

Reply:
I think just picking one language will do fine for this test. If there were edges to test, then it would make sense to test those edges, but in this case, just ensuring that the choose succeeds should be enough.

Reply:
Ok, I will test in English then. Thank you.